maven/spec/fixtures/poms/multimodule_pom.xml

```diff
@@ -49,6 +49,12 @@
                 <artifactId>util</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>net.evenh.multimodule</groupId>
+                <artifactId>util</artifactId>
+                <version>${project.version}</version>
+                <classifier>sources</classifier>
```

Comment:
Could you add a test case in the filer_parser spec covering this case? Does this change mean we correctly detect internal dependencies?

Reply:
There is already a test case for this. Adding this classified dependency should cover the change. And yes, without the patch this added dependency will fail a test case where it asserts all dependencies to look for updates.

Reply:
https://github.com/dependabot/dependabot-core/blob/main/maven/spec/dependabot/maven/file_parser_spec.rb#L725 should be the test case.

Reply:
Oh gotcha! Thanks for linking 👍